### ISSUE TITLE
Is Ktweak still relevant? A11 Oneplus Nord N100 here

### ISSUE THREAD

**Issue by BD9Max** [Magisk-Modules-Alt-Repo/ktweak#35]
?

**Comment by 1dopewrx** [comment]
No, I have tested a few phones with the SD8 Gen 1, Gen 2 and Gen 2 for Galaxy (the OC'd one). At this point all it achieved was keeping cpu clocks pinned. You can just get a kernel manager and have more success than with this with today's phones. It used to make a massive difference back in the 860,865 days.
